src/pretix/base/migrations/0072_auto_20170803_1006.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.2 on 2017-08-03 10:06
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def set_priority(app, schema_editor):
+    wles = app.get_model('pretixbase', 'WaitingListEntry')
+    for wle in wles.objects.all():
+        if wle.voucher:
+            wle.priority = None
+        else:
+            wle.priority = 10
+        wle.save()
```

Comment:
Thanks for the neat suggestions to reduce database hit count. I've learned much! 😄